test-infra/cmdExportOutput.sh

```diff
@@ -192,7 +192,7 @@ echo "SKIP_GATEWAY_AWS=$SKIP_GATEWAY_AWS"
 echo "SKIP_GATEWAY_GCP=$SKIP_GATEWAY_GCP"
 echo "SKIP_GATEWAY_OCI=$SKIP_GATEWAY_OCI"
 echo "SKIP_GATEWAY_AWSGOV=$SKIP_GATEWAY_AWSGOV"
-echo "SKIP_GEO_VPN=$SKIP_GEO_VPN"
```

Comment:
why remove this?

Reply:
These should be no change to cmdExportOutput.sh. Will change back.